### ISSUE TITLE
Ability to see currently online players

### ISSUE THREAD

**Issue by rhymu8354** [rhymu8354/OmniaRegna#77]
This is in-game ticket #248: [Kenkomatix]

Mabye by a UI list or something or a /list command who knows. But now where the map gets bigger it would be nice to see who is there :D

**Comment by domsson** [comment]
Agreed. In a simpler form, an indication of how many players are online would already be great. Something like

> Currently _n_ players online
